src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+interface Props {
+  error: boolean;
+  changeError: (value: boolean) => void;
+}
+
+const ErrorNotification:React.FC<Props> = ({ error, changeError }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={
+        error
+          ? 'notification is-danger is-light has-text-weight-normal'
+          : 'notification is-danger is-light has-text-weight-normal hidden'
+      }
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally